packages/@okta/vuepress-site/docs/reference/api/apps/index.md

```diff
@@ -224,6 +224,142 @@ curl -v -X POST \
 }
 ```
 
+#### Add Okta Org2Org application
+
+Adds a new Okta Org2Org application to your organization
+
+##### Settings
+
+
+| Parameter          | Description                                             | DataType | Nullable | Unique | Validation                                |
+| ------------------ | ------------------------------------------------------- | -------- | -------- | ------ | ----------------------------------------  |
+| acsURl  | The Source org Assertion Consumer Service URL | String | TRUE    | FALSE  | [URL](http://tools.ietf.org/html/rfc3986) |
+| audRestriction  | Audience URI | String | TRUE    | FALSE  | [URL](http://tools.ietf.org/html/rfc3986) |
+| baseUrl | The login URL of the target Okta Org | String  | TRUE    | FALSE  | [URL](http://tools.ietf.org/html/rfc3986) |
```

Comment:
The login URL **for** the target Okta **o**rg (small o)

[These edits could actually go back into the help text for the UI - let me know if you want to do that]